build.js

```diff
@@ -96,8 +97,8 @@ import * as rollup from 'rollup';
     },
   });
   await es8bundle.write({
-    file: 'asmcrypto.all.es8.js',
-    format: 'es',
+    file: 'asmcrypto.all.es8.cjs',
+    format: 'cjs',
   });
```

Comment:
Why is this needed, if we want to use Node.js's built-in support for ES modules? :sweat_smile:

Reply:
Noted :)